CardinalKit-Example/CardinalKit.xcodeproj/project.pbxproj

```diff
@@ -869,6 +873,7 @@
 				GCC_WARN_UNUSED_VARIABLE = YES;
 				IPHONEOS_DEPLOYMENT_TARGET = 13.0;
 				MTL_ENABLE_DEBUG_INFO = YES;
+				NEW_SETTING = "";
```

Comment:
Do we need this new config?

Reply:
I think this may be for the chartView, if not I believe we can take NEW_SETTING out